src/GitHook/Command/FileCommand/PreCommit/PhpMd/PhpMdCheckConfiguration.php

```diff
@@ -34,4 +36,34 @@ public function getMinimumPriority(): int
 
         return (isset($this->config['priority'])) ? $this->config['priority'] : $defaultPriority;
     }
+
+    /**
+     * @return string
+     */
+    public function getRulesetPath(): string
+    {
+        return $this->config['rulesetPath'] ?? $this->getDefaultRulesetPath();
+    }
+
+    /**
+     * @throws \Exception
+     *
+     * @return string
+     */
+    protected function getDefaultRulesetPath(): string
+    {
+        $defaultRuleSetPathsByPriority = [
+            'config/phpmd-ruleset.xml',
+            'vendor/spryker/spryker/Bundles/Development/src/Spryker/Zed/Development/Business/PhpMd/ruleset.xml',
+            'vendor/spryker/development/src/Spryker/Zed/Development/Business/PhpMd/ruleset.xml',
```

Comment:
Maybe this can be set on a class level so that it's possible to extend this. What do you think?